app/pages/preview/template.njk

```diff
@@ -18,6 +20,14 @@
     {{ viewFeatures({
       section: sections['features']
     }) }}
+    {{ viewIntegrations({
+      section: {
+        sections: sections
+      }
+    }) }}
+    {{ viewImplementationTimescales({
+      section: sections['implementation-timescales']
+    }) }}
```

Comment:
Are these in the right place? as in should they be after client application types? not sure what the wireframes suggest